src/System.Memory/tests/ReadOnlySpan/IndexOfAny.char.cs

```diff
@@ -321,11 +321,276 @@ public static void MakeSureNoChecksGoOutOfRangeThree_Char()
             }
         }
 
+        [Fact]
+        public static void ZeroLengthIndexOfFour_Char()
+        {
+            ReadOnlySpan<char> sp = new ReadOnlySpan<char>(Array.Empty<char>());
+            ReadOnlySpan<char> values = new char[] { (char)0, (char)0, (char)0, (char)0 };
+            int idx = sp.IndexOfAny<char>(values);
+            Assert.Equal(-1, idx);
+        }
+
+        [Fact]
+        public static void DefaultFilledIndexOfFour_Char()
+        {
+            Random rnd = new Random(42);
+
+            for (int length = 0; length < byte.MaxValue; length++)
```

Comment:
What's the benefit of testing this on every length from 0 to 255?

Reply:
Duplicate of the previous `DefaultFilledIndexOfThree_Char` but extended to 4 (in this test) and 5

Every length is probably overkill; but it should exercise all the 

sequential start up loop (alignment) -> vector length loop -> sequential remainder loop

paths?